Week3/BusinessCard/app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <View
+        android:id="@+id/grayRectangle"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/colorGrayView"
+        tools:layout_editor_absoluteX="32dp"
+        tools:layout_editor_absoluteY="0dp" />
+
+    <View
+        android:id="@+id/greenRectangle"
+        android:layout_width="337dp"
+        android:layout_height="match_parent"
+        android:background="@color/colorGreenView"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0" />
```

Comment:
You don't need this line because your view's height is set to `match_parent` and it doesn't have any effect in that case. 
I guess you did your layouts in the design tab and not through XML? 
That's fine, but I would encourage you to write XML at the beginning to grasp the concepts better. Then after some time you can switch to graphical editor if you find it easier to work with.